src/Runner.Worker/Handlers/ScriptHandlerHelpers.cs

```diff
@@ -82,18 +82,23 @@ internal static (string shellCommand, string shellArgs) ParseShellOptionString(s
             }
         }
 
-        internal static string GetDefaultShellForScript(string path, Common.Tracing trace, string prependPath)
+        internal static string GetDefaultShellNameForScript(string path, Common.Tracing trace, string prependPath)
```

Comment:
NOTE: This will cause conflicts for the container hook PR, but I'd rather solve them there